Scripts/Python/xAgeSDLBoolActivatorComboSet.py

```diff
@@ -224,3 +247,22 @@ def _ValidateCombination(self):
         if ',' not in combination.value:
             PtDebugPrint("xAgeSDLBoolActComboSet._ValidateCombination():\tCombination does not have separation delimiters")
         self._combination = [int(i) for i in combination.value.split(',')]
+
+    def _CheckSolved(self):
+        if self._numCorrect == len(self._combination):
+            PtDebugPrint("xAgeSDLBoolActComboSet._CheckSolved():\tWoohoo! You have solved the puzzle!", level=kWarningLevel)
+            self._solved = True
+            if disableOnSolve.value:
+                self._butsInUse = True
+
+    def _AddToAttempt(self, actId):
+        attempt = self._attemptCombo
+        attempt.append(actId)
+        if len(attempt) > len(self._combination):
+            attempt = attempt[1:]
```

Comment:
```suggestion
        attempt = [-len(self._combination):]
```

Reply:
I had to try this out in the REPL because I wasn't sure what this would do when len(self._combination) was greater than len(attempt) but it looks like it plays nice. Thanks for the suggestion!